apps/perf-test/tasks/perf-test.js

```diff
@@ -0,0 +1,285 @@
+const cp = require('child_process');
+const fs = require('fs');
+const path = require('path');
+const generateFlamegraph = require('./flamegraph/generateFlamegraph');
+
+// A high number of iterations are needed to get visualization of lower level calls that are infrequently hit by ticks.
+// TODO: change to 5000
+const iterations = 10;
```

Comment:
Worth defaulting to `5000` but allow for CLI arg?

Reply:
Possibly. I think it'd also be useful to control which scenarios are running as most people who run locally will most likely only care about one or two.